config/locales/alchemy.it.yml

```diff
@@ -0,0 +1,924 @@
+      intro: "Questo è come l'upload di %{name} funziona"
+      step1: "Clicca il bottone 'esplora' e naviga alla tua cartella %{name}"
+      step2: "Seleziona tutti gli oggetti %{name} di cui vuoi fare l'upload"
+      step3: "Clicca il bottone 'apri'. L'upload parte!"
+      dragndrop: "Puoi anche trascinare dei file dal tuo computer sull'area sottostante."
+    external_link_notice_1: "Prego inserisci l'url completo con http:// o un protocollo simile."
+    external_link_notice_2: "Per fare riferimento ad un indirizzo interno al tuo sito, comincia il link con /."
+    female: "Femmina"
+    file: "File"
+    file_rename_error: "Errore nel modificare il nome del file."
+    files: "File"
+    from_at: "da %{by} a %{at}"
+    height: "Altezza"
+    help:
+      shortcuts:
+        "Open help window": "Apri finestra di aiuto"
+        "Close current dialog": "Chiudi finestra attiva"
+        "Open logout dialog": "Apri finestra di logout"
+        "Focus search field": "Attiva il campo di ricerca"
+        "Create a new record": "Crea un nuovo record"
+        "Show page infos": "Mostra informazioni pagina"
+        "Create new element": "Crea nuovo elemento"
+        "Edit page properties": "Modifica proprietà pagina"
+        "Reload the preview": "Ricarica la preview"
+        "Leave the page": "Lascia la pagina"
+        "Open upload form": "Apri form di upload"
+        "Select all pictures": "Seleziona tutte le immagini"
+    hide_element_content: "Nascondi il contenuto di questi elementi."
+    dashboard: "Dashboard"
+    image_alt_tag: "Alt-tag"
+    image_caption: "Caption"
+    image_name: "Name"
+    image_title: "Title-tag"
+    images: "Immagini"
+    internal_link_headline: "Scegli una pagina a cui linkare."
+    internal_link_page_elements_explanation: "In aggiunta puoi cliccare appena sotto una pagina su 'Mostra Elementi' per linkare ad un anchor di un elemento da quella pagina."
+    internal_link_page_anchors_explanation: "Alternativamente puoi linkare ad un anchor della pagina stessa."
+    "item copied to clipboard": "%{name} copiato nella clipboard"
+    "item moved to clipboard": "%{name} spostato alla clipboard"
+    "item removed from clipboard": "%{name} eliminato dalla clipboard"
+    javascript_disabled_headline: "Javascript disabilitato!"
+    javascript_disabled_text: "Javascript è richiesto per far funzionare Alchemy al meglio. Prego abilitalo nelle configurazioni del tuo browser."
+    language_code_placeholder: 'i.e. it'
+    language_does_not_exist: "Questo albero della lingua non esiste"
+    language_pages_copied: "Albero della lingua copiato con successo."
+    last_upload_only: "Solo ultimo upload"
+    left: "sinistra"
+    legacy_url_info_text: "Un link è un redirect da una vecchia URL alla nuova URL di questa pagina. Questo redirect viene gestito tramite uno <a href='https://support.google.com/webmasters/answer/93633' target='_blank'>status code 301</a>."
```

Comment:
maybe we can link the Italian guide version? https://support.google.com/webmasters/answer/93633?hl=it